http3/server.go

```diff
@@ -173,6 +173,10 @@ func (s *Server) handleConn(sess quic.Session) {
 	(&settingsFrame{}).Write(buf)
 	str.Write(buf.Bytes())
 
+	// Wait for completion of the handshake.
+	// TODO(#2311): allow 0-RTT requests.
+	<-sess.HandshakeComplete().Done()
```

Comment:
There's no real reason to wait for completion of the handshake here, other than keeping the (behavioral) diff small in this PR. I think it will be safe to remove these lines in #2311.